### ISSUE TITLE
Centos 7 dependency failure when updating to version 1.66.0

### ISSUE THREAD

**Issue by handrews42** [microsoft/vscode#146454]
When updating to VS Code 1.66, yum reports 

Error: Package: code-1.66.0-1648620680.el7.x86_64 (code)
Requires: libc.so.6(GLIBC_2.25)(64bit)

CentOS 7 has glibc.x86_64 2.17-325.el7_9, installed and is unlikely to update further through official channels.  I could theoretically upgrade eventually to CentOS 8 but my enterprise isn't quite ready for that yet.

I can still run 1.65 as this doesn't seem to have the dependency on glibc 2.25.

**Comment by rzhao271** [comment]
Duplicate of https://github.com/microsoft/vscode/issues/146390
